src/sdk/config_fs.js

```diff
@@ -378,6 +378,30 @@ class ConfigFS {
         return identity;
     }
 
+    /**
+     * get_identity_by_id_and_stat_file returns the full account/user data and stat the file:
+     * 1. try by identity path
+     * 2. if not found - try by account name with new accounts path
+     * 3. if not found - try by account name with old accounts path
+     * silent_if_missing is used only on get identity (if stat fails, it will throw an error)
+     * @param {string} id
+     * @param {string} [type]
+     * @param {{show_secrets?: boolean, decrypt_secret_key?: boolean, silent_if_missing?: boolean}} [options]
+     * @returns {Promise<Object>} 
+    */
+    async get_identity_by_id_and_stat_file(id, type, options = {}) {
+        let identity;
+        try {
+            identity = await this.get_identity_by_id(id, type, options);
```

Comment:
can you verify identity can't be undefined. and if it can please handle this case

Reply:
I added, thanks.